src/applications/facility-locator/tests/components/VaFacilitySearch.unit.spec.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import VaFacilityResult from '../../components/search-results-items/VaFacilityResult';
+import testData from '../../constants/mock-facility-data-v1.json';
+import { LocationType } from '../../constants';
+
+describe('VaFacilitySearch', () => {
```

Comment:
The name of the test (and its file) should match the component under test, so I think we should change `VaFacilitySearch`to `VaFacilityResult` here and in the filename.